crates/common_utils/src/consts.rs

```diff
@@ -13,3 +13,7 @@ pub(crate) const ALPHABETS: [char; 62] = [
 
 /// TTL for token
 pub const TOKEN_TTL: i64 = 900;
+
+// General purpose base64 engines
+pub(crate) const BASE64_ENGINE: base64::engine::GeneralPurpose =
```

Comment:
no we already have the same in /crates/router/src/consts.rs we don't need this new const. use the existing one